objects/DbConfigFactory.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const StagingDbConfig = require('./StagingDbConfig');
+const ProductionDbConfig = require('./ProductionDbConfig');
+
+class DbConfigFactory {
+  static build(environment, dbConfig) {
+    let klass = (environment === 'production') ? ProductionDbConfig : StagingDbConfig;
```

Comment:
Just wondering if these class names are communicating well the class responsibility: ProductionDbConfig is meant to be used in production, but the difference between it and StagingDbConfig aren't clear.
Maybe we`SQLDbConfig`, `YAMLDbConfig` would be more expressive names. What do you think?